### ISSUE TITLE
Publish on central ?

### ISSUE THREAD

**Issue by bric3** [weisJ/JavaNativeFoundation#6]
The github package do not allow for anonymous download which makes it difficult to use outside github (without credentials). So for example in a build-tool that downloads the zip.

It would be practical to have it on central, the lib is fairly light (~256 KiB).

**Comment by weisJ** [comment]
I do not have any use anymore for this project personally as I have transitioned to including the relevant files in projects directly. Though if I find time I can publish this on Maven central :)

**Comment by bric3** [comment]
I'm mentioning this because I've seen other libs pointing to your JNF build.

If I got it right this is replaced by some home brewed "Apple Kit", to help with notarization?

**Comment by weisJ** [comment]
Yes. As it turned out everything I needed cut be cut down to a reasonably manageable file: https://github.com/weisJ/auto-dark-mode/blob/master/macos/src/main/objcpp/JNFUtils.mm
